### PR TITLE
added installation_linux.md

### PR DESCRIPTION
**Pull request by FUTC-Coding**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ x] Read and understood (see CONTRIBUTING.md)
- [ x] Included a live link or preview screenshot
- [ x] Images are `240 x 240` [w x h]
- [ x] Resolved merge conflicts
- [ x] Included a description of my change below

# Things done in this Pull Request

- Added documentation on how to setup the website locally, on linux. You can see it here https://github.com/Tanonic/gci17fossasia/blob/gh-pages/installation_linux.md

- screenshot of website running locally https://nextcloud.tanonic.tk/index.php/s/ET0OX7fZS9yEmU1

### CODE REVIEW

**Review by rupav — COMMENTED** [review]
@Tanonic, Please post a Screenshot of the browser which shows you were successful in setting up the local server.

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Please attach screenshots also

File: installation_linux.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Local Development Setup
+
+The instructions on this page will guide you in setting up a local development
+environment in Linux.
+
+### Ruby Installation
+1. Open a Terminal and use a package manager to install ruby. For example, on Ubuntu or Debian use `sudo apt install ruby` or on Arch Linux `sudo pacman -S ruby`
```
Comment (akshatnitd) [inline installation_linux.md]: It will be ```apt-get```

File: installation_linux.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### jekyll Installation
+1. In your Terminal enter `gem install jekyll bundler`
+
+### Setting up locally
+1. Fork this repository.
+2. Use git to clone your forked repository, i.e. git clone `https://github.com/<your_username>/gci17.fossasia.org.git`
+3. Change directory to the local repository, i.e. `cd gci17.fossasia.org`. Now type `ls` to see if `Gemfile` can be seen.
+4. Type `bundle install` to install dependencies.
+5. Now open Gemfile(with any text editor e.g. `sublime` or `nano`) and then Comment `gem 'github-pages', group: :jekyll_plugins` and uncomment `#gem 'jekyll-sitemap'`
```
Comment (akshatnitd) [inline installation_linux.md]: Is this uncommenting thing required for Linux as well ? Can you please justify the reason

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Check inline comments

**Comment by FUTC-Coding** [comment]
I added a screenshot in the description and also fixed requested items earlier today @ms10398 @akshatnitd

**Comment by akshatnitd** [comment]
LGTM :+1:

**Review by akshatnitd — APPROVED** [review]

**Comment by anshumanv** [comment]
@Tanonic I think @ms10398 wants screenshots to be added with the documentation, so users can see how it looks like on successfully running it.
Please confirm @ms10398.

**Comment by FUTC-Coding** [comment]
@anshumanv could be, but the Windows documentation doesn't have any either.

**Comment by anshumanv** [comment]
Yes, I agree @Tanonic, probably we can ask the one who added windows docs too!
Anyways please @ms10398 clarify where he needs screenshots.

**Review by rupav — APPROVED** [review]
😃Good Work and thanks for the screenshot provided.

**Review by the-codingninja — APPROVED** [review]
LGTM!!

**Pull request merged**